autoload/neomake.vim

```diff
@@ -476,6 +491,19 @@ function! neomake#MakeHandler(job_id, data, event_type) abort
             endif
         endif
         let status = a:data
+        if has_key(maker, 'exit_callback')
+            try
```

Comment:
This try block is too broad. I think we should wrap only the lines that can err in a try block, and then we should only catch the errors we expect. This is a PITA in vimscript, but there are examples in this file that should point you in the right direction.

The danger in keeping this try statement too broad is that we can accidentally hide errors that we actually want to fix, which makes it difficult to track down when they cause a bug.